README.md

```diff
@@ -34,7 +34,7 @@ This shareable configuration performs the following actions:
 ## Install
 
 ```bash
-$ npm install --save-dev semantic-release@15 @carbonaut/release-docs
+$ npm install --save-dev semantic-release@17 @carbonaut/release-docs
```

Comment:
Do we need to install `semantic-release@17`. It is already on the devDependencies of this project, so in theory `npm i` would install it automatically.

Reply:
We have two situations here:

- `semantic-release` is on the `devDependencies` from `@carbonaut/release-docs` project because this project is a NPM package. In order to deploy a NPM package we also use `semantic-release`, but it's a totally different setup.

- `semantic-release` needs to be on `devDependencies` on the project that will be using `@carbonaut/release-docs` because in this case we are using a [shareable configuration](https://semantic-release.gitbook.io/semantic-release/usage/shareable-configurations) feature from `semantic-release`. In order to setup a shareable configuration we have a `.releaserc.json` file on the project's root folder:

```
{
  "extends": "@carbonaut/release-docs"
}
```

and to execute this we need to run `semantic-release` command in the project's root folder.

Reply:
I'm still a bit confused , shouldn't `semantic-release` be a `dependency` instead of a `devDependency`?

Reply:
Just for the record, as we discussed on Slack, it's required to install `semantic-release@17` on the project that is using `@carbonaut/release-docs`.